qt/aqt/addons.py

```diff
@@ -1132,6 +1136,164 @@ def download_addons(
 ######################################################################
 
 
+class ChooseAddonsToUpdateList(QListWidget):
+    ADDON_ID_ROLE = 101
+
+    def __init__(
+        self,
+        parent: QWidget,
+        mgr: AddonManager,
+        updated_addons: List[UpdateInfo],
+    ) -> None:
+        QListWidget.__init__(self, parent)
+        self.mgr = mgr
+        self.updated_addons = sorted(
+            updated_addons, key=lambda addon: addon.suitable_branch_last_modified
+        )
+        self.ignore_check_evt = False
+        self.setup()
+        self.setContextMenuPolicy(Qt.CustomContextMenu)
+        qconnect(self.itemClicked, self.on_click)
+        qconnect(self.itemChanged, self.on_check)
+        qconnect(self.itemDoubleClicked, self.on_double_click)
+        qconnect(self.customContextMenuRequested, self.on_context_menu)
+
+    def setup(self) -> None:
+        header_item = QListWidgetItem("", self)
+        header_item.setFlags(Qt.ItemFlag(Qt.ItemIsUserCheckable | Qt.ItemIsEnabled))
+        header_item.setBackground(Qt.lightGray)
+        self.header_item = header_item
+        for update_info in self.updated_addons:
+            addon_id = update_info.id
+            addon_meta = self.mgr.addon_meta(str(addon_id))
+            update_enabled = addon_meta.update_enabled
+            addon_name = addon_meta.human_name()
+            update_timestamp = update_info.suitable_branch_last_modified
+            update_time = datetime.fromtimestamp(update_timestamp)
+
+            addon_label = f"{update_time:%Y-%m-%d}   {addon_name}"
+            item = QListWidgetItem(addon_label, self)
+            # Not user checkable because it overlaps with itemClicked signal
+            item.setFlags(Qt.ItemFlag(Qt.ItemIsEnabled))
+            print(update_enabled)
```

Comment:
This may have been missed

Reply:
Sorry about that! Thought I checked for prints before pushing.